packages/docmanager-extension/src/index.ts

```diff
@@ -1014,6 +1038,11 @@ namespace Private {
    */
   export let id = 0;
 
+  /**
+   * An array of activated file names used as a naive implemetation of lock for Name File
+   */
+  export let activatedDocs: string[] = [];
```

Comment:
Maybe this could be moved above as a local variable of the plugin, since it seems to get reassigned?

Reply:
agree, thanks :) @jtpio